spec.html

```diff
@@ -4229,14 +4228,12 @@ <h1>
         <h1>
           PutValue (
             _V_: unknown,
```

Comment:
I think we can change `unknown` to `a Reference Record or an ECMAScript language value`.

Reply:
Done.

In fact I'm pretty sure static semantic rules will prevent calling this with an ES language value, but auditing that is tricky.